scripts/tests.sh

```diff
@@ -130,21 +130,13 @@ function download_aleth()
     elif [ -z $CI ]; then
         ALETH_PATH="aleth"
     else
-        # Any time the hash is updated here, the "Running compiler tests" section should also be updated.
         mkdir -p /tmp/test
-        if grep -i trusty /etc/lsb-release >/dev/null 2>&1
-        then
-            # built from d661ac4fec0aeffbedcdc195f67f5ded0c798278 at 2018-06-20
-            ALETH_BINARY=aleth_2018-06-20_trusty
-            ALETH_HASH="54b8a5455e45b295e3a962f353ff8f1580ed106c"
-        else
-            # built from d661ac4fec0aeffbedcdc195f67f5ded0c798278 at 2018-06-20
-            ALETH_BINARY=aleth_2018-06-20_artful
-            ALETH_HASH="02e6c4b3d98299885e73f7db6c9e3fbe3d66d444"
-        fi
-        ALETH_PATH="/tmp/test/aleth"
-        wget -q -O $ALETH_PATH https://github.com/ethereum/cpp-ethereum/releases/download/solidityTester/$ALETH_BINARY
-        test "$(shasum $ALETH_PATH)" = "$ALETH_HASH  $ALETH_PATH"
+        # Any time the hash is updated here, the "Running compiler tests" section should also be updated.
+        ALETH_HASH="b044543c71d24a5a62d5f0d3e71cb1e3593560ee"
+        wget -q -O /tmp/test/aleth.tar.gz https://github.com/ethereum/aleth/releases/download/v1.5.0-alpha.5/aleth-1.5.0-alpha.5-linux-x86_64.tar.gz
```

Comment:
I think its better to leave the `ALETH_BINARY` or rather `ALETH_VERSION` variable for readability.